Deeploy/TilingExtension/CodeTransformationPasses/TilingCodeGeneration.py

```
@@ -23,150 +23,209 @@
+                opReprUpdates[i].append(snippet.operatorRepresentation)
 
-        tilePtrBuffer.allocTemplate = NodeTemplate("")
-        tilePtrBuffer.deallocTemplate = NodeTemplate("")
-        tilePtrBuffer.initTemplate = NodeTemplate("""
-        ${type.referencedType.typeName} bu_${name} = 0;
-        ${type.referencedType.typeName}* ${name} = &bu_${name};""")
-
-        return newPtrName
+        tiledSnippets: List[CodeSnippet] = [
+            CodeSnippet(*self._tileTemplate(ctxt, opReprUpdate, template, tileIdxVar, f"{tensorName}_"))
+            for template, opReprUpdate in zip(templates, opReprUpdates)
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)